### PR TITLE
feat: better slots

### PR DESCRIPTION
**Pull request by IllusionMan1212**
need help making sensible multipliers and chances

### CODE REVIEW

File: src/commands/games/Slots.ts [hunk 1]
```diff
@@ -3,19 +3,142 @@ import { Command } from "@src/Command";
 import { getCoins, subCoins, addCoins } from "@database/economy";
 import { embedify } from "@utils/embedify";
 import { createNewMember } from "@src/database/users";
+import { randomItem, randRange, reserialize } from "svcorelib";
```
Comment (Sv443) [inline src/commands/games/Slots.ts:6]: pls fix linter warnings

File: src/commands/games/Slots.ts [hunk 2]
```diff
@@ -119,33 +244,126 @@ export class Slots extends Command
...
 
-        await this.editReply(int, resultStr);
-
-        const win = this.checkWin(result, grid);
+        let resultStr = "";
+        const emojis = this.SLOTS.map((slot) => {
+            return slot.emoji;
+        });
 
         if (win) {
```
Comment (Sv443) [inline src/commands/games/Slots.ts:269]: couldn't this potentially generate two winning patterns at once?

**Review by Sv443 — APPROVED** [review]
oki doik

File: src/commands/games/Slots.ts (hunk 2 above)
Comment (IllusionMan1212) [reply src/commands/games/Slots.ts:269]: it can, but idc :low_quality_pepe:
(fuck i have to actually fix this shit)

Comment (Sv443) [reply src/commands/games/Slots.ts:269]: in that case why not just roll with it and just recount the final score

Comment (IllusionMan1212) [reply src/commands/games/Slots.ts:269]: because that's not how i want it to be + which emote takes precedence or do i count both?

Comment (Sv443) [reply src/commands/games/Slots.ts:269]: both, it would be the jackpot to have a 5x5 of eggplant  
actual billionaire jeef benzos simulator

Comment (IllusionMan1212) [reply src/commands/games/Slots.ts:269]: hmmmm too lazy. either you or canny do that :low_quality_pepe:

Comment (Sv443) [reply src/commands/games/Slots.ts:269]: understandable

Comment (IllusionMan1212) [reply src/commands/games/Slots.ts:258]: i did it anyway. plz revrev

**Comment by Sv443** [comment]
bunger

File: src/commands/games/Slots.ts [hunk 3]
```diff
@@ -106,7 +218,9 @@ export class Slots extends Command
 
     async run(int: CommandInteraction): Promise<void>
     {
-        if (!int.guild?.id) return await this.reply(int, "This channel cannot be used in DMs");
+        await this.deferReply(int);
+
+        if (!int.guild?.id) return await this.editReply(int, "This channel cannot be used in DMs");
```
Comment (Sv443) [inline src/commands/games/Slots.ts:223]: lol

**Review by Sv443 — APPROVED** [review]
pig

File: src/commands/games/Slots.ts (hunk 3 above)
Comment (IllusionMan1212) [reply src/commands/games/Slots.ts:223]: Brain damage. Don't mind it

**Pull request merged**